src/components/dialog/dialog.ts

```diff
@@ -48,6 +48,8 @@ export class MDialog extends ModulVue implements PortalMixinImpl {
     public paddingBody: boolean;
     @Prop({ default: true })
     public paddingFooter: boolean;
+    @Prop()
+    public closeTitle: string;
```

Comment:
Ce n'est pas nécessaire de créer une propriété pour ça. Je ne vois pas de cas où le tooltip devrait être différent.

Reply:
Dans prisme il y a des places qu'on écrivait "Fermer le questionnaire" je me suis dis que ce serait probablement demandé un jour :)

Reply:
Je crois tout comme @setur52 qu'il serait utile d'avoir une prop closeTitle avec comme valeur par défaut `this.$i18n.translate('m-dialog:close')`